### DISCUSSION TITLE
[Feature] Add a toggle or user setting to collapse the sidebar

### DISCUSSION THREAD

**Discussion by natedawg** [immich-app/immich#5735] [Feature Request]
### The feature

The sidebar takes up a lot of space it would be great to be able to collapse it so more photos can be in view. The toggle could be on the main web ui or just a user setting would work too. I use a vertical monitor to browse the web and the sidebar takes up 1/4 of the screen. 

### Platform

- [ ] Server
- [x] Web
- [ ] Mobile

**Comment by pablaud** [comment]
Would love to see this one implemented

**Comment by echo-saurav** [comment]
Please 🙏

**Comment by redone310413** [comment]
Would love to see this one implemented

**Comment by bo0tzz** [comment]
#16768

**Reply by badbob001** [reply]
The above change auto-collapses the sidebar based on some pre-determined window sizes, so we don't really have control of when we will see the menu when viewing on a PC or a high-resolution device.
